packages/@okta/vuepress-site/docs/guides/sampleapp-oie-redirectauth/main/index.md

```diff
@@ -262,25 +262,25 @@ Instead of signing in to Okta, it is possible to route users to an external Iden
 2. Access the [Facebook App Dashboard](https://developers.facebook.com/apps).
 3. Create a Facebook app using these [instructions](https://developers.facebook.com/docs/development/create-an-app).
 
-    > **Note:** When you are creating the app, select **Consumer** as the app type.
+    > **Note:** When you’re creating the app, select **Consumer** as the app type.
 
 4. After you create the app, on the Add Products to Your App page, click **Set Up** on the **Facebook Login** tile.
 5. On the first page of the Quickstart, select **Web**.
-6. In the **Site URL** box, enter your Org URL (found in the global header located in the upper-right corner of the Admin Console). This URL can also be any URL that makes sense for your app.
+6. In the **Site URL** box, enter your org URL. This URL can also be any URL that makes sense for your app. To locate your org URL, click your username in the upper-right corner of the Admin Console. Your org URL appears in the dropdown menu.
```

Comment:
Should we use "org URL" or "Okta domain"? I normally equate URL to the full locator form with the HTTP/HTTPS scheme.